app/models/prescription.rb

```diff
@@ -22,7 +22,7 @@
 # @!attribute ordered_date
 #   @return [Vets::Type::UTCTime]
 # @!attribute quantity
-#   @return [Integer]
```

Comment:
@agravell047 Will you need a data migration to update the existing values in your db? Also, line 25 still says string. Finally, do you want to add any validation to your model?

Reply:
@agravell047 If you are storing the values in a db, and are okay with storing integers and strings in the same column, which can add complexity to retrieving the values, then no need. It